ivy/functional/ivy/elementwise.py

```diff
@@ -6369,11 +6370,13 @@ def tan(
 @to_native_arrays_and_back
 @handle_array_function
 @handle_device_shifting
+@handle_complex_input
 def tanh(
     x: Union[ivy.Array, ivy.NativeArray],
     /,
     *,
     out: Optional[ivy.Array] = None,
+    complex_mode: Literal["split", "magnitude", "jax"] = "jax",
```

Comment:
Probably some of the previous changes might've been accidentally reverted, as there doesn't seem to be a `complex_mode` argument for `ivy.tanh` in the backends 😅

Reply:
`complex_mode` is handled by a function wrapper rather than by the backends themselves

Reply:
yep sure, but ideally the `ivy.tanh` function and the backend implementations should have the same function signature (not typehints of course, but the arguments to the function)

Reply:
Yep, you're right, I'll add that in